docs/pages/2-usage.md

```diff
@@ -60,7 +60,7 @@ $ tree /tmp/data/dns/00000/
 0 directories, 1 file
 ```
 
-If you stop the container, and the data will persist on the host. If you didn't mount to the host, then stopping the container means losing the data.
```

Comment:
This should be 

>> If you stop the container and had mounted it, the data will persist on the host. If you didn't mount to the host, then you will not see the data.

It's possible to stop (and not remove a container) and then start it again with the id to get the data, but that's too complicated for the average user, so it's best to just suggest mounting the data folder if they want to keep it (or not risk losing it).